spec/predicates/link.md

```diff
@@ -1,14 +1,14 @@
 # Predicate type: Link
 
-Type URI: https://in-toto.io/Link/v0.1
+Type URI: https://in-toto.io/Link/v0.2
 
-Version: 0.1.0
+Version: 0.2.0
 
 ## Purpose
 
-A generic attestation type with the same schema as [in-toto 0.9]. This allows
-existing in-toto users to make minimal changes to upgrade to the new attestation
-format.
+A generic attestation type with the a schema isomorphic to [in-toto 0.9]. This
```

Comment:
```suggestion
A generic attestation type with a schema isomorphic to [in-toto 0.9]. This
```

Reply:
Thanks for always fixing my typos. :)

Reply:
Haha, no problem. I'm honestly not reviewing _only_ to fix typos 😬